codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/RestJson.kt

```diff
@@ -47,6 +49,44 @@ class RestJsonFactory : ProtocolGeneratorFactory<HttpBoundProtocolGenerator> {
     }
 }
 
+/**
+ * This [HttpBindingResolver] mostly delegates to [HttpTraitHttpBindingResolver].
```

Comment:
```suggestion
 * This [HttpBindingResolver] mostly delegates to [HttpBindingResolver].
```

Reply:
`HttpBindingResolver` is the interface; this class delegates to the `HttpTraitHttpBindingResolver` class. I've reworded the docs.

Reply:
Ok thanks.